dev/TabView/TabView.xaml

```diff
@@ -606,32 +620,59 @@
                                     </VisualState.Setters>
                                 </VisualState>
                             </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="AdjacencyStates">
+                                <VisualState x:Name="NotAdjacent"/>
+                                <VisualState x:Name="AdjacentOnTheLeft">
+                                    <VisualState.Setters>
+                                        <Setter Target="TabContainer.CornerRadius" Value="{ThemeResource TabViewItemAdjacentOnTheLeftCornerRadius}"/>
+                                        <Setter Target="TabSeparator.Opacity" Value="0"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="AdjacentOnTheRight">
+                                    <VisualState.Setters>
+                                        <Setter Target="TabContainer.CornerRadius" Value="{ThemeResource TabViewItemAdjacentOnTheRightCornerRadius}"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
 
-                        <Path 
-                            x:Name="LeftRadiusRender"
+                        <Path
+                            x:Name="LeftRadiusRenderArc"
+                            x:Load="False"
+                            Fill="{ThemeResource TabViewItemHeaderBorderColor}"
+                            VerticalAlignment="Bottom" Visibility="Collapsed"
+                            Margin="{ThemeResource TabViewItemLeftRadiusMargin}"
+                            Data="M4 0C4 1.19469 3.47624 2.26706 2.64582 3C1.94069 3.62236 1.01445 4 0 4V3C1.65685 3 3 1.65685 3 0H4Z"/>
+
+                        <Path
```

Comment:
Why do we need this now?

Reply:
Otherwise, how would we do it ?:)
![image](https://user-images.githubusercontent.com/21356912/111194386-627f1f00-8578-11eb-9d33-5984a1e44235.png)
We need two pieces here: an arc that will draw that little "slope" and a "triangle" that will fill the space below the slope with the background color of the selected tab

Reply:
The previous embedded SVG did that in one single shape, arc as the top left, then the left border and the bottom border and fill that.

Reply:
yes, but we can't just fill this time because we need a stroke on the outside

Reply:
Ah I see, that makes sense.